app/models/country.rb

```diff
@@ -1,4 +1,20 @@
 class Country < ActiveRecord::Base
-  belongs_to :language, :foreign_key => 'LanguageID'
+
+  # create_table :countries do |t|
```

Comment:
I appreciate the thought behind putting the schema information in comments here, but we might just want to add the [annotate models](https://github.com/ctran/annotate_models) gem which will do this automatically. I usually add it to new projects but left it out of this one until things had stabilized a bit more because it can cause unnecessary merge conflicts in these early stages (as would manually updating these large comment blocks, which would still be better than letting them get out of date). I wonder whether it makes more sense to drop these for now until the schema settles a bit more.